src/components/Home/index.js

```diff
@@ -29,13 +29,13 @@ const Home = () => {
           Dino-War
         </p>
         <AppLink
-          href="#/levels/beginner/1"
+          to="/levels/beginner/1"
           title="New Game"
         >
           New Game
         </AppLink>
         <AppLink
-          href="#/levelselect"
+          to="/levelselect"
```

Comment:
to="/levels"
just levels now would be correct. My mistake on telling you otherwise

Reply:
Fixed these links in the merged Rules PR.

Reply:
Fixed